.docker/autoline/docker-compose.dev.yml

```diff
@@ -1,74 +1,54 @@
 version: "3.8"
 
-# Using `node_modules` mounting pattern described here:
-# https://burnedikt.com/dockerized-node-development-and-mounting-node-volumes/#prevent-hosts-node_modules-from-being-mounted
-
 services:
   autoline_mariadb:
+    image: mariadb
```

Comment:
Prisma migration **won't work** on the sole `mariadb` image. Because the default image allows its user to access only one database. Prisma requires to have access to all the DBs in the system. So we must have the `user_priviligies.sql` script added here